pkg/prove/proof_test.go

```diff
@@ -19,7 +19,7 @@ import (
 func TestTxInclusion(t *testing.T) {
 	txCount := 100
 	typicalBlockData := types.Data{
-		Txs:      generateRandomlySizedContiguousShares(txCount, 200),
+		Txs:      generateRandomlySizedContiguousShares(txCount, 1000),
```

Comment:
Can this go up to 4069 or more? Or can we somehow explicitly test the case where a tx/msg spans multiple rows as well?

Reply:
yep, good idea. https://github.com/celestiaorg/celestia-core/blob/ff2bcbde0685c8a311d087514bb009e3cc9e6bac/pkg/prove/proof_test.go#L29-L47